Cyface-App/Source/Views/RegistrationView.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  RegistrationView.swift
+//  Cyface-App
+//
+//  Created by Klemens Muthmann on 18.07.22.
+//
+
+import SwiftUI
+import WebKit
+
+/**
+ A view showing the registration page on the Cyface Website
+
+ - author: Klemens Muthmann
+ - version: 1.0.0
+ */
+struct RegistrationView: UIViewControllerRepresentable {
+
+    /*@ObservedObject var model: RegistrationViewModel
+    let registrationURL: URL
+
+    init(serverAddress: String?, model: RegistrationViewModel) {
+        self.model = model
+        guard let serverAddress = serverAddress else {
+            fatalError()
+        }
+
+        guard let serverURL = URL(string: serverAddress) else {
+            fatalError()
+        }
+
+        self.registrationURL = serverURL
+    }
+
+    func makeCoordinator() -> Coordinator {
+        return Coordinator(self)
+    }
+
+    func makeUIViewController(context: Context) -> RegistrationNavigationDelegate {
+        let webViewController = RegistrationNavigationDelegate(registrationURL: registrationURL, coordinator: context.coordinator)
+        webViewController.loadUrl(registrationURL.appendingPathComponent("registration"))
+
+        return webViewController
+    }
+
+    func updateUIViewController(_ uiView: RegistrationNavigationDelegate, context: UIViewControllerRepresentableContext<RegistrationView>) {
+    }
+
+    func onRegistrationFinished() {
+        model.onRegistrationFinished()
+    }*/
```

Comment:
any reason for this commented-out code?

Reply:
Ah damn. That is a problem with XCode. If you delete a file in XCode it is not necessarily deleted in the local file system, so you end up with orphan files, that are not part of the application anymore. This whole file needs to be deleted. The correct version is in Cyface-App/Source/Views/Registration/RegistrationView.swift